Jenkinsfile_CNP

```diff
@@ -39,16 +44,89 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+// Vars needed for AKS testing
+env.TEST_S2S_NAME = 'bulk_scan_processor_tests'
+env.TEST_S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+env.TEST_SCAN_DELAY = '4000'
+env.RESOURCE_GROUP = 'bulk-scan-aks-rg'
+
 GradleBuilder builder = new GradleBuilder(this, product)
 
 withPipeline(type, product, component) {
+  //after('checkout') {
+  //  echo sh(returnStdout: true, script: 'env')
+  //}
+  after('smoketest:aks') {
```

Comment:
after smoketest:aks seems too late shouldn't it be after deploy?

Reply:
Our smoke tests don't use that stuff, so that should work. It can be moved as well, it's not a big deal.

Reply:
to avoid surprises can we move it please?, i.e. our smoketests should be doing more
unless there's a reason not to?